### PR TITLE
multiplatform: images and fonts

### PR DESCRIPTION
**Pull request by avently**

### CODE REVIEW

File: apps/multiplatform/android/src/main/java/chat/simplex/app/ui/theme/Type.kt [hunk 1]
```diff
@@ -4,16 +4,16 @@ import androidx.compose.material.Typography
...
-  Font(R.font.inter_semi_bold, weight = FontWeight.SemiBold),
-  Font(R.font.inter_medium, weight = FontWeight.Medium),
-  Font(R.font.inter_light, weight = FontWeight.Light),
+val Inter: FontFamily = FontFamily(
+  Font(MR.fonts.Inter.regular.fontResourceId),
+  Font(MR.fonts.Inter.italic.fontResourceId, style = FontStyle.Italic),
+  Font(MR.fonts.Inter.bold.fontResourceId, FontWeight.Bold),
+  Font(MR.fonts.Inter.semibold.fontResourceId, FontWeight.SemiBold),
+  Font(MR.fonts.Inter.medium.fontResourceId, FontWeight.Medium),
+  Font(MR.fonts.Inter.light.fontResourceId, FontWeight.Light)
```
Comment (epoberezkin) [inline apps/multiplatform/android/src/main/java/chat/simplex/app/ui/theme/Type.kt:16]: are we even using these fonts? I think we have some others installed

File: apps/multiplatform/common/src/commonMain/resources/distribute/SimpleX.desktop [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Name=SimpleX
+Comment=Messenger for the future
```
Comment (epoberezkin) [inline apps/multiplatform/common/src/commonMain/resources/distribute/SimpleX.desktop:4]: we need different text here

File: apps/multiplatform/android/src/main/java/chat/simplex/app/ui/theme/Type.kt (hunk 1 above)
Comment (avently) [reply apps/multiplatform/android/src/main/java/chat/simplex/app/ui/theme/Type.kt:16]: https://github.com/simplex-chat/simplex-chat/pull/2656/files#diff-f910305dc69b00400b4b17861797a27e1a98d377ff8ffcecffb291961c870fceL22

https://github.com/simplex-chat/simplex-chat/pull/2656/files#diff-f910305dc69b00400b4b17861797a27e1a98d377ff8ffcecffb291961c870fceL27

https://github.com/simplex-chat/simplex-chat/pull/2656/files#diff-79a51e6d9c4a2a89f013c84187ef8672202cddee9e2f7e7776553ec873902ffcR226

So Bold and Normal are in use at least

> I think we have some others installed

Where?

File: apps/multiplatform/common/src/commonMain/resources/distribute/SimpleX.desktop (hunk 2 above)
Comment (avently) [reply apps/multiplatform/common/src/commonMain/resources/distribute/SimpleX.desktop:4]: Like what?

File: apps/multiplatform/android/src/main/java/chat/simplex/app/ui/theme/Type.kt (hunk 1 above)
Comment (spaced4ndy) [reply apps/multiplatform/android/src/main/java/chat/simplex/app/ui/theme/Type.kt:16]: Inter is the font we use

**Pull request merged**